packages/babel-helper-remap-async-to-generator/src/index.js

```diff
@@ -62,6 +62,11 @@ function plainFunction(path: NodePath, callId: Object) {
 
   node.async = false;
   node.generator = true;
+  // Either the wrapped generator is invoked with `.apply(this, arguments)` or it has no params,
+  // so it should capture `arguments`
+  if (node.shadow) {
+    node.shadow = { arguments: false };
```

Comment:
Added some info to the wiki https://github.com/babel/babel/wiki/Internal-Plugins